### PR TITLE
Recent Envoy changes + bug fixes

### PR DESCRIPTION
**Pull request by mikepmanning**
Changes

### CODE REVIEW

File: doc/index.rst [hunk 1]
```diff
@@ -29,6 +29,10 @@ This system is what was previously called m&m, which is no longer available. m&m
 Installing
 -----------------------------
 
+Download the system you want to use. If you use classflexing, the Allclass system will enable everything but it has it's kinks. If you plan on sticking to one class, the class system will be the one you download. DO NOT EXTRACT THE SYSTEM. After downloading, open Mudlet and your profile and open the Package Manager. Click install and select the .zip system that you downloaded. It will ask you to select where the m&m file is, you'll want to choose where your profile folder is saved as the file will be there.
```
Comment (vadi2) [inline doc/index.rst:32]: Changing zip to mpackage would fix this

Comment (mikepmanning) [reply doc/index.rst:32]: Is it as simple as naming it .mpackage instead of .zip?

**Comment by vadi2** [comment]
For Mudlet, yeah. You'll want to change documentation and such to reflect
new package names.

File: doc/index.rst (hunk 1 above)
Comment (mikepmanning) [reply doc/index.rst:32]: I'll try to add that for the next release - I'll get this one out for the latest batch of envoy changes

**Pull request merged**